File: Modules/Common/Items/SIE.Web.Items/ProductModels/Commands/LineCapacityAddCommand.cs
using SIE.Web.Command;

namespace SIE.Web.Items.ProductModels.Commands
{
    /// <summary>
    /// 添加
    /// </summary>
    public class LineCapacityAddCommand : ViewCommand
    {
        /// <summary>
        /// 执行
        /// </summary>
        /// <param name="args">args</param>
        /// <param name="scope">scope</param>
        /// <returns>执行结果</returns>
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}

File: Modules/Common/Items/SIE.Web.Items/ProductModels/Commands/LineCapacityAddCommand.js
SIE.defineCommand('SIE.Web.Items.ProductModels.Commands.LineCapacityAddCommand', {
    extend: 'SIE.cmd.Add',
    meta: { text: "添加".t(), group: "edit" },
    getEditEntity: function () {
        var newEntity = new this.view._model();
        var parent = this.view.getParent();
        var productModelId = parent.getCurrent().data.Id;
        if (this.view.isListView) {
            newEntity = this.createNewItem();
        }
        this.onItemCreated(newEntity);
        newEntity.data.ProductModelId = productModelId;
        return newEntity;
    },
});